go-controller/pkg/metrics/master.go

```diff
@@ -485,16 +489,32 @@ const (
 	// port binding with updated chassis seen in OVN-Kubernetes  control plane southbound database libovsdb cache
 	portBindingChassis
 	portBindingTable = "Port_Binding"
+
+	queueCheckPeriod           = time.Millisecond * 500
+	addPortBinding   operation = iota
+	updatePortBinding
+	addPod
+	cleanPod
+	addLogicalSwitchPort
 )
 
 type record struct {
 	timestamp time.Time
 	timestampType
 }
 
+// item is the data structure for ControlPlaneRecorders queue
+type item struct {
+	op     operation
+	old    *model.Model
+	new    *model.Model
```

Comment:
Why the pointers? A Model is already a pointer.

Reply:
Ouch, didnt know this - thanks! Will remove.